include/gillwald/geometry.hpp

```diff
@@ -6,13 +6,19 @@
 #include <vector>
 
 namespace geometry {
+
+	struct Cell {
```

Comment:
Can you add doxygen comments like `bresenham_conversion` has?

Reply:
Also, add member comments like
```
int x;  ///< The x coordinate of the cell.
int y;  ///< The y coordinate of the cell.
```

Reply:
Done